Binary files xorg-server-ubuntu-git-patched/.git/index and xorg-server-ubuntu-git-virtual/.git/index differ
diff -Nurp xorg-server-ubuntu-git-patched/hw/xfree86/common/xf86AutoConfig.c xorg-server-ubuntu-git-virtual/hw/xfree86/common/xf86AutoConfig.c
--- xorg-server-ubuntu-git-patched/hw/xfree86/common/xf86AutoConfig.c	2009-08-24 14:39:45.000000000 -0700
+++ xorg-server-ubuntu-git-virtual/hw/xfree86/common/xf86AutoConfig.c	2009-08-24 14:54:37.000000000 -0700
@@ -168,6 +168,7 @@ videoPtrToDriverList(struct pci_device *
 	case 0x1002:		    driverList[0] = "ati"; break;
 	case 0x102c:		    driverList[0] = "chips"; break;
 	case 0x1013:		    driverList[0] = "cirrus"; break;
+        case 0x1234:                driverList[0] = "vesa"; break; /* qemu device not supported by cirrus */
 	case 0x8086:
 	    switch (dev->device_id)
 	    {
